src/stored-atom.ts

```diff
@@ -0,0 +1,29 @@
+import { Atom } from './atom';
+import { Observable } from 'rxjs';
+import { identity } from 'ramda';
+
+interface StoredAtomOptions<T> {
+  key: string;
+  from?: (source: any) => T,
+  to?: (value: T) => any
+}
+
+export class StoredAtom<T> extends Atom<T> {
+  private storedValue$: Observable<T>;
+
+  constructor(defaultValue: T, {key, from = identity, to = identity}: StoredAtomOptions<T>) {
+    const savedItem = localStorage.getItem(key);
+    super(savedItem === null ? defaultValue : from(JSON.parse(savedItem)));
```

Comment:
I'd say this one: `from(JSON.parse(savedItem))` should be called `parse` and this `SON.stringify(to(value))` - `serialize`. They may be just functions in constructor scope, but named ones. If you do that expression like `savedItem === null ? defaultValue : parse(savedItem)` is clear enough to have no name.